src/components/__tests__/change-view-test.jsx

```diff
@@ -0,0 +1,379 @@
+        it('sets state.diffType to the stored value', () => {
+          const storedDiffType = 'CHANGES_ONLY_TEXT';
+          layeredStorage.getItem.mockReturnValue(storedDiffType);
+
+          const changeView = shallow(
+            <ChangeView
+              page={simplePage}
+              from={{content_type: 'text/html'}}
+              to={{content_type: 'text/html'}}
+              user={{email: 'me'}}
+            />,
+            {context: {api: mockApi}}
+          );
+
+          expect(changeView.state().diffType).toBe(storedDiffType);
+        });
+      });
+
+      describe('when that diffType is NOT relevant to the pages being compared', () => {
+        describe('when defaultDiffType (SIDE_BY_SIDE_RENDERED) is relevant to the pages being compared', () => {
+          it('sets state.diffType to SIDE_BY_SIDE_RENDERED', () => {
+            const storedDiffType = 'IRRELEVANT_DIFF_TYPE';
+            layeredStorage.getItem.mockReturnValue(storedDiffType);
+
+            const changeView = shallow(
+              <ChangeView
+                page={simplePage}
+                from={{content_type: 'text/html'}}
+                to={{content_type: 'text/html'}}
+                user={{email: 'me'}}
+              />,
+              {context: {api: mockApi}}
+            );
+
+            expect(changeView.state().diffType).toBe(defaultDiffType);
+          });
+        });
+
+        describe('when defaultDiffType (SIDE_BY_SIDE_RENDERED) is NOT relevant to the pages being compared', () => {
+          it('sets state.diffType to the first relevant diff type', () => {
+            const storedDiffType = 'IRRELEVANT_DIFF_TYPE';
+            layeredStorage.getItem.mockReturnValue(storedDiffType);
+
+            const mediaType = 'text/*';
+            const relevantTypes = diffTypesFor(mediaType);
+
+            const changeView = shallow(
+              <ChangeView
+                page={simplePage}
+                from={{content_type: mediaType}}
+                to={{content_type: mediaType}}
+                user={{email: 'me'}}
+              />,
+              {context: {api: mockApi}}
+            );
+
+            expect(changeView.state().diffType).toBe(relevantTypes[0].value);
+          });
+        });
+      });
+    });
+
+    describe('when a diffType has NOT been stored in layeredStorage', () => {
+      describe('when defaultDiffType (SIDE_BY_SIDE_RENDERED) is relevant to the pages being compared', () => {
+        it('sets state.diffType to SIDE_BY_SIDE_RENDERED', () => {
+          layeredStorage.getItem.mockReturnValue(null);
+
+          const changeView = shallow(
+            <ChangeView
+              page={simplePage}
+              from={{content_type: 'text/html'}}
+              to={{content_type: 'text/html'}}
+              user={{email: 'me'}}
+            />,
+            {context: {api: mockApi}}
+          );
+
+          expect(changeView.state().diffType).toBe('SIDE_BY_SIDE_RENDERED');
```

Comment:
Should this be `defaultDiffType` instead of the actual string `SIDE_BY_SIDE_RENDERED`?